pkg/networkservice/utils/count/server.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package count
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+
+	"github.com/golang/protobuf/ptypes/empty"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+
+	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+)
+
+// Server is a server type for counting Requests/Closes
+type Server struct {
+	Requests, Closes int32
```

Comment:
Can we add methods for these variables?

Something like:
`Requests() int32` and `Closes() int32` 


In this case it will be consistent with `UniqueRequests` and `UniqueCloses` api.

Moreover it will make code more simple, imagine you will not more use `atomic.LoadInt32(&counter.Closes)` in each count check :)

Reply:
Yep, done.